src/library.js

```diff
@@ -447,7 +447,7 @@ LibraryManager.library = {
   // and this function _abort(). Remove one of these, importing two functions for the same purpose is wasteful.
   abort__sig: 'v',
   abort: function() {
-    abort();
```

Comment:
Can we make `-O3` and `-Oz` builds use the old code-size optimized form?

Reply:
Oops, yes I meant to do that.

Reply:
Ah yes, I just forgot to upload by list revision... I had already put behind ASSERTIONS.